### PR TITLE
Introductory page for hardware/accessories

### PR DESCRIPTION
**Pull request by hamishwillee**
Creates hardware page to introduce the main hardware/accessory options and explain "common" top level accessory-relevant information (e.g. safety issues, keepout zones, etc)

### CODE REVIEW

**Comment by hamishwillee** [comment]
@dmoroni This is the page you reviewed last week. @mrpollo This should be mergable.

**Comment by mrpollo** [comment]
merged master back to this tree, if tests are ok we can merge

File: book/starting-accessories.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+In order to communicate with Solo your accessory must establish a connection and be recognised by Solo. The two supported interfaces are USB and I2C (depending on payload bay).
+
+Information for communicating with each bay is part of individual bay documentation.
+
+
+<aside class="note">
+At time of writing you can connect via USB but care needs to be taken not to interfere with other port configuration and devices.
+
+We are currently developing a framework for safe and seamless accessory discoverability, integration with apps, and update.
```
Comment (peterbarker) [inline book/starting-accessories.md]: s/update/updates/

We are currently developing a framework for safe and seamless accessory discoverability, app integration, and firmware updates. 
?

Comment (hamishwillee) [inline book/starting-accessories.md]: I like that.

**Comment by hamishwillee** [comment]
@peterbarker Thanks for review. I fixed that typo (accepted your improved sentence). Ready to merge.

**Pull request merged**